src/Serilog.Sinks.Console/Sinks/SystemConsole/Formatting/ThemedJsonValueFormatter.cs

```diff
@@ -248,7 +248,7 @@ int FormatLiteralValue(ScalarValue scalar, TextWriter output)
             }
 
             using (ApplyStyle(output, ConsoleThemeStyle.Scalar, ref count))
-                JsonValueFormatter.WriteQuotedJsonString(value.ToString(), output);
+                JsonValueFormatter.WriteQuotedJsonString(value.ToString() ?? "null", output);
```

Comment:
As above, in this case if `ToString()` returns `null`, `""` may be slightly more accurate.

Reply:
That is a hard one. I think if someone _on purpose_ inheritance a ToString() to return `null`, The code should deal with the same way that the instance was `null`. But if you prefer to be `""` i don't see any problems. That should be extremely rare to happen.
I would let you decide.

Reply:
Thanks for the reply 👍 

I think it's a bit scenario-dependent; imagine some code like:

```csharp
var x = SomeClass.SomeOpaqueMethod();
Log.Information("Loaded {X}", x);
```

I think in this case, personally, I'd want to know whether `x` is null, in the log event, and not what its `ToString()` method happened to return.

Reply:
The only problem that I see is that I think we need to keep consistency in all Serilog Projects. 
And I fix this in Serilog Core using `null`. https://github.com/serilog/serilog/pull/1427 and https://github.com/serilog/serilog/pull/1452

Should I change all to return `"",` in `Serilog` and in `Serilo.Skink.Console`?

Reply:
That's a great point - sorry I missed this the first time around. I think we should be consistent and use `""` when `ToString()` returns null, if possible :+1: